supportedsites.md

```diff
@@ -1541,7 +1541,6 @@
  - **VGTV**: VGTV, BTTV, FTV, Aftenposten and Aftonbladet
  - **vh1.com**
  - **vhx:embed**: [*vimeo*](## "netrc machine")
- - **Viafree**
```

Comment:
Revert. This will be auto-generated.

Reply:
Thank you for letting me know, I was not aware :upside_down_face: